docs/octopus-rest-api/octopus.server.exe-command-line/database.md

```diff
@@ -39,3 +39,9 @@ Or one of the common options:
       --help                 Show detailed help for this command
 ```
 
+## Basic example
+This example will create a new database for the MyNewInstance instance.  This example expects that the database for instance MyNewInstance doesn't exist, if it does, it will say it already exists and won't do anything.
```

Comment:
```suggestion
This example creates a new database for the `MyNewInstance` instance.  This example expects the database for instance `MyNewInstance` doesn't exist, if it does, it will say it already exists and won't do anything:
```